CodeGen/templates/Action.h

```diff
@@ -101,18 +106,19 @@ struct {{data.ModuleAPI}} F{{data.StructName}}GetResultResponse
 	GENERATED_BODY()
 
 public:
-	int8 status;
+	int8 Status;
```

Comment:
```suggestion
        UPROPERTY(EditAnywhere, BlueprintReadWrite)
	int8 Status = 0;
```

Reply:
int8 can't be use with bp

Reply:
Ah right, but it could still go with UPROPERTY(), just for it to be reset in PIE replay

Reply:
OK, I created following pr to add UPROPERTY()
https://github.com/rapyuta-robotics/UE_tools/pull/6